site/SiteLogos.scss

```diff
@@ -33,14 +33,9 @@
                 opacity: initial;
             }
         }
-        .oxford-logo img {
```

Comment:
The viewboxes in the SVGs are set such that the borders in each graphic will all be flush when the SVG elements are the same height.